src/serializer/ResidualHeapVisitor.js

```diff
@@ -230,6 +230,10 @@ export class ResidualHeapVisitor {
     }
   }
 
+  visitValueReactElement(val: ObjectValue): void {
+    // TODO: do we even need to visit anything on the React element specifically?
```

Comment:
Probably not. Test cases would help to determine this, as elsewhere we have the invariant that all visited values must be serialized.

Reply:
In fact, it seems like you want to visit *less* for ReactElements. Consider tweaking `visitObjectProperties`.